### PR TITLE
En_Daiku_Kakariko OK (Kakariko Village Carpenters)

### PR DESCRIPTION
**Pull request by Zelllll**

### CODE REVIEW

File: src/overlays/actors/ovl_En_Daiku_Kakariko/z_en_daiku_kakariko.c [hunk 1]
```diff
@@ -43,29 +58,464 @@ static ColliderCylinderInit D_809E54B0 = {
     },
     { 18, 66, 0, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Daiku_Kakariko/func_809E4320.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Daiku_Kakariko/EnDaikuKakariko_Init.s")
+static CollisionCheckInfoInit2 sColChkInit = { 0, 0, 0, 0, 0xFF };
+
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Daiku_Kakariko/z_en_daiku_kakariko.c:63]: MASS_IMMOBILE for ff

File: src/overlays/actors/ovl_En_Daiku_Kakariko/z_en_daiku_kakariko.c [hunk 2]
```diff
@@ -43,29 +58,464 @@ static ColliderCylinderInit D_809E54B0 = {
...
+    if ((*currentAnimIndex < 0) || (animIndex == *currentAnimIndex)) {
+        transitionRate = 0.0f;
+    } else {
+        transitionRate = sAnimations[animIndex].transitionRate;
+    }
+
+    Animation_Change(&this->skelAnime, sAnimations[animIndex].animation, 1.0f, 0.0f,
+                     Animation_GetLastFrame(sAnimations[animIndex].animation), sAnimations[animIndex].mode,
+                     transitionRate);
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Daiku_Kakariko/z_en_daiku_kakariko.c:82]: transistionrate is now called morphFrames. there may be some other animation related changes you need to make for temp vars

**Comment by Zelllll** [comment]
alright fixed all of fig's comments

**Review by fig02 — APPROVED** [review]

**Comment by Zelllll** [comment]
Ok I fixed everything that you mentioned, should be good to merge

**Review by Roman971 — APPROVED** [review]

**Pull request merged**